src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/ModuleBuilderImpl.cs

```diff
@@ -15,8 +15,10 @@ internal sealed class ModuleBuilderImpl : ModuleBuilder
         private readonly string _name;
         private Type?[]? _coreTypes;
         private readonly Dictionary<Assembly, AssemblyReferenceHandle> _assemblyRefStore = new();
+        private readonly Dictionary<ConstructorInfo, MemberReferenceHandle> _constructorRefStore = new();
```

Comment:
> Is the "store" prefix common in this area

Not sure, the prefix came from the prototype, I can change the name

> This should be eventually used to map any member references to tokens. It can be Dictionary<MemberInfo, MemberReferenceHandle>. (ConstructorInfo inherits from MemberInfo.)

Makes sense